<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnatomyCarve</class>
 <widget class="qMRMLWidget" name="AnatomyCarve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>749</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="dataCollapsableButton">
     <property name="text">
      <string>AnatomyCarve Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_intensityVolume">
        <property name="text">
         <string>Intensity volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="intensityVolume">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true">Singleton</string>
        </property>
        <property name="selectNodeUponCreation">
         <bool>true</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>intensityVolume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_segmentation">
        <property name="text">
         <string>Segmentation:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="segmentation">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true">Singleton</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>segmentation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_view">
        <property name="text">
         <string>View:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="view">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLViewNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true">Singleton</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>view</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="qMRMLSegmentsTableView" name="segmentsClipping">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_sphereRadius">
        <property name="text">
         <string>Sphere radius:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkSliderWidget" name="sphereRadius">
        <property name="toolTip">
         <string>Sets the radius of the invisible carving sphere.</string>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="minimum">
         <double>50.000000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="value">
         <double>200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_segmentsClipping">
        <property name="text">
         <string>Clipping for last added sphere:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_clippingSphere">
        <property name="text">
         <string>Clipping spheres:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="renderButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Render</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="qSlicerSimpleMarkupsWidget" name="clippingSpheres">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>5</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="nodeSelectorVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerSimpleMarkupsWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerSimpleMarkupsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentsTableView</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentsTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>AnatomyCarve</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>intensityVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AnatomyCarve</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentation</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AnatomyCarve</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>view</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>segmentation</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>segmentsClipping</receiver>
   <slot>setSegmentationNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
